pkg/components/velero/component_test.go

```diff
@@ -170,6 +170,7 @@ component "velero" {
     backup_storage_location {
       bucket   = "foo"
       provider = "aws"
+      region   = "myregion"
```

Comment:
Nit: this change makes test pass, but we don't have a test where configuration error is expected.